packages/backend-app-api/src/http/readHelmetOptions.test.ts

```diff
@@ -47,7 +47,7 @@ describe('readHelmetOptions', () => {
       csp: {
         key: ['value'],
         'img-src': false,
-        'script-src-attr': ['custom'],
+        scriptSrcAttr: ['custom'],
```

Comment:
Could we keep a test that makes sure we still support the old format too? Maybe with a deprecation warning too?

Reply:
ops good catch